.evergreen/run-csfle-tests-with-mongocryptd.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
```

Comment:
This script was created from `run-tests.sh` by removing pieces that are not relevant (based on eyeballing and experimenting) and changing the Gradle tasks and test filters.